tests/acceptance/10_files/12_acl/acl_dir.cf

```diff
@@ -1,42 +0,0 @@
-# These tests don't work
-# body file control
-# {
-#     cache_system_functions => "false";
-# }
-
-bundle agent main
-{
-  files:
-    "$(filenames[1])"
-      create => "true",
-      acl => default;
-
-  vars:
-    "N"  slist  =>  { "1" };
-
-    "filenames[1]"  string  => "/tmp/acl_default.tmpdir/.";
-
-    "expected[1]" string => ".*user::rwx.*group::rw-.*other::r--.*default:user::rwx.*default:group::rw-.*default:other::r--.*";
-
-    "output[$(N)]" string => execresult("getfacl -cp $(filenames[$(N)]) | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/ /g'", "useshell");
-
-
-  classes:
-    "regcmp_matched_$(N)" expression => regcmp("$(expected[$(N)])", "$(output[$(N)])");
-
-  reports:
-      "FAILED TO MATCH CASE $(N)"
-        unless => "regcmp_matched_$(N)";
-      "EXPECTED for $(N):    $(expected[$(N)])";
-#        unless => "regcmp_matched_$(N)";
-      "GOT for $(N)     :    $(output[$(N)])";
-#        unless => "regcmp_matched_$(N)";
-}
-
-body acl default
-{
-    acl_method => "overwrite";
-    aces =>  { "all:r:allow", "user:*:rwx:allow", "group:*:rw:allow" };
-    acl_type => "generic";
-    acl_default => "access";
-}
```

Comment:
You add this file in one commit, and then remove it in another. Maybe don't add it at all then?